sql/_32_damson/cbrd_22783_REGEXP_functions/cases/REGEXP_SUBSTR.sql

```diff
@@ -0,0 +1,95 @@
+--mandatory argument
+SELECT REGEXP_SUBSTR('Samseong-ro 86-gil, Gangnam-gu, Seoul', ',[^,]+,');
+SELECT REGEXP_SUBSTR();
+SELECT REGEXP_SUBSTR ('abc');
+SELECT REGEXP_SUBSTR ('aabbb', 'ab{-1,3}$');
+SELECT REGEXP_SUBSTR('', ',[^,]+,');
+SELECT REGEXP_SUBSTR('Samseong-ro 86-gil, Gangnam-gu, Seoul', '');
+SELECT REGEXP_SUBSTR(NULL, ',[^,]+,');
+SELECT REGEXP_SUBSTR('Samseong-ro 86-gil, Gangnam-gu, Seoul', NULL);
+
+--optional arguments
+SELECT REGEXP_SUBSTR('Samseong-ro 86-gil, Gangnam-gu, Seoul', '\w+\-\w+', 1, 1);
+SELECT REGEXP_SUBSTR('Samseong-ro 86-gil, Gangnam-gu, Seoul', '\w+\-\w+', 1, 'ab');
+SELECT REGEXP_SUBSTR('Samseong-ro 86-gil, Gangnam-gu, Seoul', '\w+\-\w+', 1, 0);
+SELECT REGEXP_SUBSTR('Samseong-ro 86-gil, Gangnam-gu, Seoul', '\w+\-\w+', 1, -2);
+SELECT REGEXP_SUBSTR('Samseong-ro 86-gil, Gangnam-gu, Seoul', '\w+\-\w+', 1, 4);
+SELECT REGEXP_SUBSTR('Samseong-ro 86-gil, Gangnam-gu, Seoul', '\w+\-\w+', 1, NULL);
+SELECT REGEXP_SUBSTR('Samseong-ro 86-gil, Gangnam-gu, Seoul', '\w+\-\w+', 1, 1);
+SELECT REGEXP_SUBSTR('Samseong-ro 86-gil, Gangnam-gu, Seoul', '\w+\-\w+', 1, 'ab');
+SELECT REGEXP_SUBSTR('Samseong-ro 86-gil, Gangnam-gu, Seoul', '\w+\-\w+', 1, 0);
+SELECT REGEXP_SUBSTR('Samseong-ro 86-gil, Gangnam-gu, Seoul', '\w+\-\w+', 1, -2);
+SELECT REGEXP_SUBSTR('Samseong-ro 86-gil, Gangnam-gu, Seoul', '\w+\-\w+', 1, 4);
+SELECT REGEXP_SUBSTR('Samseong-ro 86-gil, Gangnam-gu, Seoul', '\w+\-\w+', 1, NULL);
+SELECT REGEXP_SUBSTR('Samseong-ro 86-gil, Gangnam-gu, Seoul', '\w+\-GU', 1, 1, 'i');
+SELECT REGEXP_SUBSTR('Samseong-ro 86-gil, Gangnam-gu, Seoul', '\w+\-GU', 1, 1, 'c');
+SELECT REGEXP_SUBSTR('Samseong-ro 86-gil, Gangnam-gu, Seoul', '\w+\-GU', 1, 1, '');
+SELECT REGEXP_SUBSTR('Samseong-ro 86-gil, Gangnam-gu, Seoul', '\w+\-GU', 1, 1, 'A');
+SELECT REGEXP_SUBSTR('Samseong-ro 86-gil, Gangnam-gu, Seoul', '\w+\-GU', 1, 1, NULL);
+
+--functional test
+--;lin on
```

Comment:
Do we need it?

Reply:
';lin on' is specified in csql. So it does not recognize it during the test.